src/main/resources/org/jenkins/plugins/lockableresources/LockStep/config.jelly

```diff
@@ -17,11 +17,14 @@
     <f:checkbox title="${%entry.inversePrecedence.checkbox.title}"/>
   </f:entry>
   <f:entry field="skipIfLocked">
-    <f:checkbox title="${%entry.inversePrecedence.skipIfLocked.title}"/>
+    <f:checkbox title="${%entry.skipIfLocked.title}"/>
   </f:entry>
   <f:entry title="${%entry.resourceSelectStrategy.title}" field="resourceSelectStrategy">
     <f:textbox/>
   </f:entry>
+  <f:entry title="${%entry.priority.title}" field="priority">
+    <f:number/>
+  </f:entry>
```

Comment:
I think we can enable the combination as follow

```
            if (inversePrecedence) {
                if (priority != 0) {
                    queueIndex = 0;
                } else {
                    priority = priority * -1;
                }
            }

```

in that case you can combine it as well.
I think it will be sure what happens. Just inverse your priority.
And it will be much more easy in code, as try to catch all the 'invalid' combination and explain it somehwere in the docu.
@PayBas  what you mean?

Reply:
I think inverting the priority value using `priority = priority * -1` will be extremely confusing unless documented very clearly.
I also think it will not be very useful for most situations.

Personally I would use something like:
```
if (inversePrecedence && priority == 0) {
    queueIndex = 0;
} else {
...
```
This will effectively _disable_ `inversePrecedence` from doing anything if any non-0 priority is set.

Then you can write in the help text of `inversePrecedence`:
> This option is ignored if "priority" is specified.

Reply:
> Despite claiming to be incompatible: the "Pipeline Syntax" builder will happily combine them and generate:
> 
> ```
> lock(inversePrecedence: true, priority: -1000000000, resourceSelectStrategy: '') {
>     // some block
> }
> ```
> 
> This doesn't show any warning.
> 

> In fact:
> 
> > Inverse precendense is not combinable with priority !
> 
> My testing indicates that it in fact _is_ compatible to some extent. In the case of 2 jobs with _identical_ queue priority, the one with inverse precedense will be placed higher in the queue.

Hm, it shall be not possible to set _prio_ && _inversePrecedense_ at the same time
https://github.com/jenkinsci/lockable-resources-plugin/blob/fb77678539d072c211e3679c78a24390177a6c82/src/main/java/org/jenkins/plugins/lockableresources/LockStepResource.java#L122C1-L124C10
And I provide exact test case for that
https://github.com/jenkinsci/lockable-resources-plugin/blob/fb77678539d072c211e3679c78a24390177a6c82/src/test/java/org/jenkins/plugins/lockableresources/LockStepTest.java#L1716

I will recheck the pipeline syntax again. This is strange, may you pls tell me, which Jenkins version you use?